### DISCUSSION TITLE
EST server in EJBCA

### DISCUSSION THREAD

**Discussion by venkataravikumaralladi** [Keyfactor/ejbca-ce#489] [Q&A] [answered]
Hi,
I am planning to EJBCA as PKI server, and want to use EST protocol. I have a device which has EST client implementation and want to create/renew certificate request from device to EST server. How we can configure EJBCA to act as EST server ? Documentation link will be helpful. Thanks

**Comment by Realiserad** [comment, marked as answer]
Unfortunately, support for [EST](https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/protocols/est) is an enterprise feature in EJBCA.

**Reply by primetomas** [reply]
https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/protocols/est
